tgg/relatedness/n_non_ref_pairs.py

```diff
@@ -203,17 +200,14 @@ def get_samples_n_non_ref(
     logger.info(
         "Retrieving pairwise shared %i non ref sample sites...", non_ref_samples
     )
-    mt = hl.vds.read_vds(vds_path).variant_data
-    mt = mt.filter_cols(~hl.literal(control_samples).contains(mt.s))
     ht = get_n_non_ref_sites(
         vds_path=vds_path,
         temp_path=temp_path,
         non_ref_samples=non_ref_samples,
         het_only=het_only,
+        control_samples=control_samples,
     )
-    mt = mt.annotate_rows(**ht[mt.row_key])
-    mt = mt.filter_rows(hl.is_defined(mt.ac))
-    ht = get_and_count_sample_pairs(mt, temp_path, non_ref_samples)
+    ht = get_and_count_sample_pairs(ht)
```

Comment:
ooh, sorry, I totally missed that the mt going into `get_and_count_sample_pairs` wasn't filtered to adj in my last review